packages/connect-core/src/entities/Repo.ts

```diff
@@ -1,27 +1,18 @@
+import { resolveMetadata, resolveManifest } from '../utils/metadata'
 import {
   AragonArtifact,
   AragonArtifactRole,
   AragonManifest,
-  ConnectionContext,
   Metadata,
+  RepoData,
 } from '../types'
-import { resolveMetadata, resolveManifest } from '../utils/metadata'
 import Organization from './Organization'
 
-export interface RepoData {
-  address: string
-  artifact?: string | null
-  contentUri?: string
-  manifest?: string | null
-  name: string
-  registry?: string
-  registryAddress?: string
-}
-
 export default class Repo {
   #metadata!: Metadata
   readonly address: string
   readonly contentUri?: string
+  readonly lastVersion?: string
```

Comment:
Had to include the `lastVersion` argument as we need it on one of the new helper functions.